### PR TITLE
25.0.0 beta 1

### PR DESCRIPTION
**Pull request by skjnldsv**
@nickvergessen @AndyScherzinger @PVince81 @nextcloud/server

### CODE REVIEW

File: config/config.php [hunk 1]
```diff
@@ -569,6 +569,38 @@
 	],
 
 	'beta' => [
+		'25' => [
+			'100' => [
+				'latest' => '25.0.0 beta 1',
```
Comment (PVince81) [inline config/config.php:574]: "beta 1" or "beta1" ? do we have guidelines ? past record ?

Comment (skjnldsv) [reply config/config.php:574]: It's taken from the server version.php 
https://github.com/nextcloud/server/blame/cd95fce105fe5f0e71b1bcac7685464f936b9749/version.php#L36

Comment (skjnldsv) [reply config/config.php:574]: It's `25.0.0 RC1`, `25.0.0 dev`, `25.0.0 beta 1` and  `25.0.0`

**Review by PVince81 — APPROVED** [review]
:+1:

**Review by AndyScherzinger — APPROVED** [review]

**Pull request merged**